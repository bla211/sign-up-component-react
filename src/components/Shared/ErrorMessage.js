import React, { Component } from 'react';
import PropTypes from 'prop-types';

class ErrorMessage extends Component {
    render(){
        const { errorMessage } = this.props;

        return(
            <div className="error">{ errorMessage }</div>
        )
    }

}

ErrorMessage.propTypes = {
    errorMessage: PropTypes.string
  };

export default ErrorMessage;