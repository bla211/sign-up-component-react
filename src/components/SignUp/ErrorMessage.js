import React, { Component } from 'react';
import PropTypes from 'prop-types';

class ErrorMessage extends Component {
    render(){
        const { errorMessage } = this.props;

        return(
            <div class="error">{ errorMessage }</div>
        )
    }

}

ErrorMessage.propTypes = {
    errorMessage: PropTypes.string
  };

export default ErrorMessage;