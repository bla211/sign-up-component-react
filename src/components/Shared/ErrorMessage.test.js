import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ErrorMessage from './ErrorMessage';

configure({ adapter: new Adapter() });

describe('Test ErrorMessage component', () => {
    test('Disclaimer renders properly when passed "disclaimerMessage"', () => {
        const errorMessage = "*error message";
        const wrapper = shallow(<ErrorMessage errorMessage={errorMessage}/>);
        expect(wrapper).toMatchSnapshot();
    });
});