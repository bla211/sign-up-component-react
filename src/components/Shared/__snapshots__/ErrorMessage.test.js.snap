// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ErrorMessage component Disclaimer renders properly when passed "disclaimerMessage" 1`] = `
<div
  className="error"
>
  *error message
</div>
`;
